crates/nu-parser/src/parser.rs

```diff
@@ -290,7 +290,7 @@ pub fn parse_external_call(
     for span in &spans[1..] {
         let contents = working_set.get_span_contents(*span);
 
-        if contents.starts_with(b"$") || contents.starts_with(b"(") {
```

Comment:
I'd keep this condition as it was before.

Reply:
Ok, done for that